### ISSUE TITLE
Interactive Shell

### ISSUE THREAD

**Issue by sebalix** [OCA/odoorpc#5]
_From @arkhan on November 10, 2016 18:6_

Hello,

Have future plans to create an interactive shell like erppeek

thanks

_Copied from original issue: osiell/odoorpc#34_

**Comment by sebalix** [comment]
Yes this is a feature I would like to see in a future version, but I can not tell you a date ;)  Anyway it is a good idea.

**Comment by sebalix** [comment]
_From @arkhan on November 11, 2016 12:51_

Great, I'll be looking forward

**Comment by sebalix** [comment]
_From @arkhan on July 13, 2017 17:43_

This is still within the TODOS

**Comment by sebalix** [comment]
Yes it is.
I just started the development, you can follow it here: https://github.com/sebalix/odoorpc/commits/interactive-shell
Not yet usable, but I hope to have enough time to implement it. You're very welcome to test it from the above branch and give some feedback.

To install OdooRPC from this development branch:
```sh
$ sudo pip install git+https://github.com/sebalix/odoorpc.git@interactive-shell
```
Then run the `odoorpc` command:
```sh
$ odoorpc
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
(Shell)
>>> odoorpc
<module 'odoorpc' from '/mnt/data/dev/odoorpc/odoorpc/__init__.pyc'>
>>> ODOO
<class 'odoorpc.odoo.ODOO'>
>>>
```

Later I'll add command line parameters to instantiate automatically an Odoo session, help and user documentation, but the first step is to ensure that this feature can be correctly tested on Python 2&3 on Travis.

Regards,

**Comment by arkhan** [comment]
Any updatev?
